src/api/interceptor.ts

```
@@ -28,56 +31,49 @@ axiosInstance.interceptors.request.use(
-    //   }
-    // }
+    if (status === HTTP_STATUS_CODE.UNAUTHORIZED) {
+      try {
+        const newAccessToken = await getRefreshToken();
+
+        store.dispatch(tokenActions.setToken(newAccessToken));
+        originalRequest.headers.Authorization = `Bearer ${newAccessToken}`;
+
+        return await axiosInstance(originalRequest);
+      } catch (err) {
+        delete axiosInstance.defaults.headers.Authorization;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)